org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java

```diff
@@ -90,7 +90,8 @@ public enum XMLSchemaErrorCode implements IXMLErrorCode {
 	TargetNamespace_2("TargetNamespace.2"), //
 	SchemaLocation("SchemaLocation"), //
 	schema_reference_4("schema_reference.4"), //
-	src_element_3("src-element.3");
+	src_element_3("src-element.3"), //
+	catalog_uri("catalog_uri");

```

Comment:
The `catalog_uri` error is not an error coming from XML validation based on XSD. Please remove it and create a `XMLCatalogErrorCode` in the catalog extensions